docs/imports/rock-the-vote.md

```diff
@@ -102,27 +102,49 @@ If an existing user has a null `mobile` profile field and provides a phone numbe
 - We do **not** override an existing phone number.
 - We save a user's mobile number regardless of whether they opt-in to SMS messaging from DS.
 
-### SMS Status
+### Email Subscription
 
-If an existing user **opts-in** to voting-related SMS messaging from DS via RTV form, we update `sms_status` as `active` if current value is either null, `less`, `stop`, or `undeliverable`.
+Email subscriptions are not updated.
 
-If an existing user **opts-out** of voting-related SMS messaging from DS via RTV form, we update `sms_status` as `stop` if current value is null or `undeliverable`.
+### SMS Subscription
 
-### SMS Subscription Topics
+If an existing user **opts-in** to voting-related SMS messaging from DS:
 
-If an existing user **opts-in** to voting-related SMS messaging from DS via RTV form, we add the `voting` topic to their `sms_subscription_topics` if it doesn't exist.
+- add the `voting` topic to `sms_subscription_topics` if it doesn't exist.
 
-If an existing user **opts-out** of voting-related SMS messaging from DS via RTV form, we remove the `voting` topic from their `sms_subscription_topics` if it exists.
+- update `sms_status` as `active` if current value is either null, `less`, `stop`, or `undeliverable`.
```

Comment:
This is very much out of this PR's scope (and my contextual knowledge) but this Q occurred to me while reading this. If a user has the 'general' `sms_subscription_topics`, and we update their status here from, say, `null`. Are we opting them into something more then what they intended in VR world?

Reply:
We shouldn't ever have a scenario where a user has a `general` subscription topic but a null `sms_status` value. The plan is backfill all current SMS subscribers with both `general` and `voting` subscription topics. Anyone who will be opting-in via RTV who was previously unsubscribed will only receive the `voting` subscription topic. Does this answer your question?

Reply:
Ahh this does indeed, thank you!! (I guess we remove the topics if someone unsubscribes entirely?)